feat(starr): Add tag option to `How to Sync 2 Radarr/Sonarr with each other` (#2372)
TRaSH-Guides/Guides: docs/Radarr/Tips/Sync-2-radarr-sonarr.md

![!Radarr - Settings Download Clients Categories](images/radarr-settings-download-clients-categories.png)

The only thing you need to change/create is a separate `Category` that you are going to use for {++Radarr 2++}

> !!! warning "This can not and should not be the same category you use for ^^==Radarr 1==^^, but a separate `Category`."

Reviewer: ```suggestion
!!! warning "This must not be the same category you use for ^^==Radarr 1==^^ - use a new `Category`."
```

### How does this Radarr Sync work (using full sync)

You now only need to make use of ^^==Radarr 1==^^. All movies you add to ^^==Radarr 1==^^ will be synced to {++Radarr 2++} at a scheduled time.

---

## Option 2 (profiles)

!!! info ""

    In this example, we will use two Sonarr instances (The basics are the same for Radarr).

    - ^^==Sonarr 1==^^ = The main 1080p WEB-DL Sonarr.
    - {++Sonarr 2++} = 4K/2160p (UHD) WEB-DL Sonarr.

>     This option will show how to cherry-pick which movies/TV shows you want as 4K/2160p (UHD).

Reviewer: ```suggestion
    This option will show how to selectively add which movies/TV shows you want as 4K/2160p (UHD).
```

![!Sonarr Categories](images/sonarr-categories.png)

The only thing you need to change/create is a separate `Category` that you are going to use for {++Sonarr 2++}

> !!! warning "This can not and should not be the same category you use for ^^==Sonarr 1==^^, but a separate `Category`."

Reviewer: ```suggestion
!!! warning "This must not be the same category you use for ^^==Sonarr 1==^^, - use a new `Category`."
```

    1. If you use multiple conditions, you can choose which condition is required, in this example, we enabled it.

    After you add all the needed info and click on save, you will see the new condition you created.

    ![Auto Tagging - Add Condition - Final Steps](images/radarr-auto-tagging-add-condition-final-steps.png)

    1. The name used for the `Auto Tagging`, in this example, we used `Radarr-UHD => HD`.
    1. If you want to remove the tags automatically if the condition no longer matches.
    1. The tags we will use in {++Radarr 2++}. In this example, we will use the tag: `radarr-hd-sync`.
    1. The condition you created before.
    1. Do not forget to click on `Save`.

=== "3rd party"

    **3rd party apps**

    You can use several third-party apps for this option. In this example, we will use Jellyseerr (_The same concept also works with Overseerr_).

    In Jellyseerr, go to `Settings` => `Services`, where you will add Radarr/Sonarr (_If you already added Radarr/Sonarr, click on edit_).

    !!! info "If you have multiple Radarr added, make sure you edit it in ^^==Radarr 1==^^"

>     ![Tags - Using a 3rd party apps - Jellyseerr](images/radarr-tags-3rd-party-app-jellyseerr.png)

Reviewer: ```suggestion
    ![Tags - Using a 3rd party app - Jellyseerr](images/radarr-tags-3rd-party-app-jellyseerr.png)
```
